libandroid-navigation/src/main/java/org/maplibre/navigation/android/Json.kt

```diff
@@ -0,0 +1,8 @@
+package org.maplibre.navigation.android
+
+import kotlinx.serialization.json.Json
+
+val json = Json {
+    ignoreUnknownKeys = true
+    explicitNulls = false
```

Comment:
We might want to add `isLenient = true` as well :)

Reply:
I have no opinion to this, does that have any downides? If not, I will add this as well 👍

Reply:
I think this increases the interop, so I'd vote for it.

> Removes JSON specification restriction (RFC-4627) and makes parser more liberal to the malformed input. In lenient mode, unquoted JSON keys and string values are allowed.

https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/-json-builder/is-lenient.html

Reply:
I enabled the `isLenient` option.